INN/umbrella-citylimits
# Special Projects Secondary Nav

## Changes

This pull request makes the following changes:

Branched off of #88 

- Adds a new `Secondary navigation` metabox to the landing page editor
- Displays the chosen secondary navigation on the landing page and as a dropdown on mobile

Desktop:
![Screen Shot 2019-11-06 at 4 08 56 PM](https://user-images.githubusercontent.com/18353636/68338109-cbff3b80-00af-11ea-9eed-a993e0b4dd17.png)

Mobile closed:
![Screen Shot 2019-11-06 at 4 09 06 PM](https://user-images.githubusercontent.com/18353636/68338107-cbff3b80-00af-11ea-99b3-b833332f4d3b.png)

Mobile open:
![Screen Shot 2019-11-06 at 4 09 11 PM](https://user-images.githubusercontent.com/18353636/68338105-cbff3b80-00af-11ea-9de8-60ab56731ac8.png)

## Why



For #69

## Testing/Questions

Features that this PR affects:

- Special projects landing pages


Questions that need to be answered before merging:


Steps to test this PR:

1. Add a secondary navigation menu to a landing page
2. Make sure landing page template is set to special projects template
3. Verify it looks ok on desktop
4. Verify it looks ok + toggles ok on mobile
